glean/lang/lsif/Glean/LSIF/Driver.hs

```diff
@@ -0,0 +1,93 @@
+{-
+  Copyright (c) Meta Platforms, Inc. and affiliates.
+  All rights reserved.
+
+  This source code is licensed under the BSD-style license found in the
+  LICENSE file in the root directory of this source tree.
+-}
+
+{-
+
+Generic indexer for LSIF. Supply name of lsif binary in $PATH to run
+predicates for typescript.
+
+-}
+
+{-# LANGUAGE OverloadedStrings #-}
+
+module Glean.LSIF.Driver ( runIndexer, processLSIF, writeJSON ) where
+
+import Control.Exception ( throwIO, ErrorCall(ErrorCall) )
+import System.Directory ( makeAbsolute, withCurrentDirectory )
+import System.FilePath
+    ( addExtension, (</>), dropExtension, takeBaseName )
+import System.IO.Temp ( withSystemTempDirectory )
+import System.Process ( callProcess )
+import Text.Printf ( printf )
+
+import qualified Data.Aeson as A
+import qualified Data.Aeson.Types as A
+import qualified Data.ByteString as B
+import qualified Data.ByteString.Lazy as Lazy
+import Data.Text ( Text )
+import qualified Data.Text as Text
+import qualified Data.Text.IO as Text
+import qualified Data.Vector as V
+
+import Data.LSIF
+import qualified Data.LSIF.Angle as LSIF ( toAngle )
+
+import Util.Log ( logInfo )
+import Util.Text ( textShow )
+import qualified Foreign.CPP.Dynamic
+import qualified Data.Aeson as Aeson
```

Comment:
you imported `Data.Aeson` twice (are we using `-Wall` in the OSS build? If not there's going to be a lot of cleaning up to do when we import this)

Reply:
Import ordering: `Data.LSIF` should come below these